app/models/select_facet.rb

```diff
@@ -6,6 +6,17 @@ def initialize(facet)
     @allowed_values = facet.allowed_values
   end
 
+  def options
+    allowed_values.map do | allowed_value |
+      {
+        "value" => allowed_value['value'],
```

Comment:
`allowed_value` should be an object here so you can call `allowed_value.value`, etc.